designer/client/conditions/select-conditions.js

```diff
@@ -99,19 +101,20 @@ class SelectConditions extends React.Component {
                   }
                 </select>
               </div>
+            }
+            {!inline &&
               <div className='govuk-form-group'>
                 <a href='#' id='inline-conditions-link' className='govuk-link'
                   onClick={this.onClickDefineCondition}>Define
-                    a new condition</a>
+                  a new condition</a>
               </div>
-            </div>
             }
             <Flyout title='Define condition' show={inline}
               onHide={this.onCancelInlineCondition}>
               <InlineConditions data={this.props.data} path={this.props.path}
                 conditionsChange={this.onSaveInlineCondition}
                 cancelCallback={this.onCancelInlineCondition}
-                hideAddLink={this.props.data.hasConditions} />
+                hideAddLink />
```

Comment:
just remove this prop declaration if you're not passing anything into it

Reply:
FYI these aren't actually equivalent. Specifying the parameter here is treated as hideAddLink={true} in the called component, whereas removing is is treated as hideAddLink={false}.

In this case though the hideAddLink has become superfluous since all callers specify it, so I've made it's behaviour the default one!

Reply:
you should be passing in `hideAddLink={true}` then, it's not clear what passing in nothing does if I'm just looking at select-conditions.js. the constructor should be detecting whether or not hideAddLink is being passed in and handling that in the component

Reply:
Fair enough. I've just pushed a fix that removes the flag entirely so that'll take care of that!